torch/onnx/_internal/exporter.py

```diff
@@ -75,14 +82,179 @@ class ONNXFakeContext:
+        """
+        for aten_name, aten_overloads_func in torchlib_registry.items():
+            internal_name_instance = registration.OpName.from_qualified_name(aten_name)
+            for overload_func in aten_overloads_func.overloads:
+                symbolic_function = registration.SymbolicFunction(
+                    onnx_function=overload_func,
+                    op_full_name=internal_name_instance.qualified_name(),
+                    is_custom=False,
+                    is_complex=False,
+                )
+                self._register(internal_name_instance, symbolic_function)
+
+            for complex_func in aten_overloads_func.complex:
+                symbolic_function = registration.SymbolicFunction(
+                    onnx_function=complex_func,
+                    op_full_name=internal_name_instance.qualified_name(),
+                    is_custom=False,
+                    is_complex=True,
+                )
+                self._register(internal_name_instance, symbolic_function)
+
+    @_beartype.beartype
+    def _register(
+        self,
+        internal_qualified_name: registration.OpName,
+        symbolic_function: registration.SymbolicFunction,
+    ) -> None:
+        """Registers a SymbolicFunction to an operator.
+
+        Args:
+            internal_qualified_name: The qualified name of the operator to register: OpName.
+            symbolic_function: The SymbolicFunction to register.
+        """
+        self._registry[internal_qualified_name].append(symbolic_function)
+
+    @_beartype.beartype
+    def register_custom_op(
+        self,
+        function: Union["onnxscript.OnnxFunction", "onnxscript.TracedOnnxFunction"],
+        namespace: str,
+        op_name: str,
+        overload: Optional[str] = None,
+        is_complex: bool = False,
+    ) -> None:
+        """Registers a custom operator: torch.ops.<namespace>.<op_name>.<overload>.
+
+        Args:
+            function: The onnx-sctip function to register.
+            namespace: The namespace of the operator to register.
+            op_name: The name of the operator to register.
+            overload: The overload of the operator to register. If it's default overload,
+                leave it to None.
+            is_complex: Whether the function is a function that handles complex valued inputs.
+
+        Raises:
+            ValueError: If the name is not in the form of 'namespace::op'.
+        """
+        internal_name_instance = registration.OpName.from_name_parts(
+            namespace=namespace, op_name=op_name, overload=overload
+        )
+        symbolic_function = registration.SymbolicFunction(
+            onnx_function=function,
+            op_full_name=internal_name_instance.qualified_name(),
+            is_custom=True,
+            is_complex=is_complex,
+        )
+        self._register(internal_name_instance, symbolic_function)
+
+    @_beartype.beartype
+    def get_functions(
```

Comment:
```suggestion
    def get_op(
```

That is tricky. Although we want the onnx function under the hood, we registered that function using `register_op` (focus on `_op`), so fetching that function with `get_op` seems more intuitive.

users can add op, check whether the op is register and get the op. the function term seems secondary

Reply:
Use `get_op_functions` instead. I think list of functions in return is still worths to mention.